README.md

```diff
@@ -1 +1,33 @@
 # DSCI-310_predicting-shares_group-4
+
+## Predicting the Shares of News Articles based on Social networks
+
+### Group members:
+
+Amar Gill, Anshnoor Kaur, Hanyu Dai, Yanxin Liang
+
+### Overview
+
+Use dataset from open source websites [UCI Machine Learning Repository: Data Sets](https://archive.ics.uci.edu/dataset/332/online+news+popularity) to predict the number of shares of a article. We split the data into traing and testing parts, and proceed full model and reduced model to make the results more reliable and efficent enough.
+
+### Dependencies
+
+To finish up this project you will need to install:
+
+-   R programming language
+-   R packages:
+    -   `renv`
+    -   `GGally`
```

Comment:
Renv will deal with the packages, so only R and Quarto are needed.

Reply:
ok, i will do this